worker-macros/src/event.rs

```diff
@@ -48,12 +48,12 @@ pub fn expand_macro(attr: TokenStream, item: TokenStream) -> TokenStream {
 
             let error_handling = match respond_with_errors {
                 true => {
-                    quote! {
+                    quote! { 
                         ::worker::Response::error(e.to_string(), 500).unwrap().into()
                     }
                 }
                 false => {
-                    quote! { panic!("{}", e) }
+                    quote! { e.into() }
```

Comment:
Because we should remove the `From<Error>` for `Response`, we should go back to panicking.